core/src/main/resources/jenkins/fingerprints/GlobalFingerprintConfiguration/config_pt_BR.properties

```diff
@@ -21,3 +21,4 @@
 # THE SOFTWARE.
 
 Fingerprints=Assinaturas
+Disable\ Fingerprint\ Cleanup=---TranslateMe --- Disable\ Fingerprint\ Cleanup
```

Comment:
Is this intended to be like that?

Reply:
Nope, it's not. I missed this one.
I guess using a simple git hook will help me avoiding those cases.
But I just fixed this one too.